pkg/gadgets/trace/dns/tracer/tracer.go

```diff
@@ -295,16 +295,24 @@ func parseLabelSequence(sample []byte) (ret string) {
 	return ret
 }
 
-func parseDNSEvent(rawSample []byte) (nameserver string, ret string, pktType string, qType string) {
+func parseDNSEvent(rawSample []byte) (qr string, nameserver string, ret string, pktType string, qType string) {
```

Comment:
What about making `parseDNSEvent` return directly `types.Event`? Or use another struct for this?